app/src/main/java/com/example/mealsnap_prototype_1/ConfigActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.example.mealsnap_prototype_1;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import services.authhandler.Amplify;
+
+public class ConfigActivity extends AppCompatActivity {
+
+    //Config activity will config stuff we need to(passes or succeeds activity just redirects with intents to what we wanna check).
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_config);
+
+        //Test Amplify Config
+        Amplify testAmplify = new Amplify();
+        testAmplify.configure(getApplicationContext());
+
+
+        //Redirect
+        goToSignupActivity();
+    }
+
+    //moves to Signup
+    private void goToSignupActivity() {
```

Comment:
Can we rename this to `loadMainAppActivity()` or something similar? Signup Activity is very specific and I would imagine this would change to the onboarding screen. Also, in the case of if the user is logged in, it should automatically redirect them to the feed activity. These are not things we need to handle immediately but all those logic should be handle by a more general function.

Reply:
Acknowledged in recent push